### DISCUSSION TITLE
Any option to bulk-delete "Archive"-photos via CLI?

### DISCUSSION THREAD

**Discussion by magicdude4eva** [photoprism/photoprism#1493] [Q&A] [answered]
Thanks for the great work on Photoprism - I have started moving my library (about 9000 photos) from iPhoto to my Synology and am currently organising the library. I have moved a large number of unwanted pictures into Archive and would like to remove all of them in bulk. 

I know that there is an upcoming feature and I would be more than happy to sponsor your work (I already bought the app which I think is great) but right now features such as multiple selects, delete and bulk-edit (especially faces, location information etc) would make this perfect.

**Comment by graciousgrey** [comment, marked as answer]
You can use a shift click / long touch to select multiple photos and then delete them.
As you mentioned batch edit (which will include faces, tags, locations etc) is already on our roadmap: #271 

What app did you buy, do you mean PhotoSync?

**Comment by magicdude4eva** [comment]
Thanks for that - I will then wait for the features to become available. At the moment it is extremely cumbersome to single-select images. I am now doing bulk-updates via the DB. I do think you guys have made a fantastic product considering that you compete with bigger players. FWIW - I bought PhotoSync - loving it and will use it to sync new images.
